astropy/io/fits/convenience.py

```diff
@@ -634,7 +637,7 @@ def append(filename, data, header=None, checksum=False, verify=True, **kwargs):
             f = _File(filename, mode='append')
             try:
                 hdu._output_checksum = checksum
-                hdu._writeto(f)
+                hdu.writeto(f)
```

Comment:
Why changing this ?

Reply:
Sorry, I was testing it and forgot to change it back... will do so A.S.A.P.